captum/attr/_core/neuron/neuron_feature_ablation.py

```diff
@@ -214,7 +225,7 @@ def neuron_forward_func(*args):
                 )
                 return _verify_select_column(layer_eval[0], neuron_index)
 
-        ablator = FeatureAblation(neuron_forward_func)
+        ablator: FeatureAblation = FeatureAblation(neuron_forward_func)
```

Comment:
It isn't necessary to have a type hint here, it can be automatically inferred by the type checker.